Libplanet/Blockchain/BlockChain.cs

```diff
@@ -264,38 +276,6 @@ public Transaction<T> GetTransaction(TxId txId)
             }
         }
 
-        /// <summary>
-        /// Returns a <see cref="long"/> integer that represents the number of elements in the
-        /// <see cref="BlockChain{T}"/>.
-        /// </summary>
-        /// <returns>A number that represents how many elements in the <see cref="BlockChain{T}"/>.
-        /// </returns>
-        public long LongCount() => Store.CountIndex(Id);
-
-        public void Validate(
```

Comment:
Has this been removed to prevent iterating access to the whole chain?

Reply:
Yes.